apps/dao/src/store/createProposalsSlice.ts

```diff
@@ -477,19 +477,50 @@ const createProposalsSlice = (set: SetState<State>, get: GetState<State>): Propo
 
           await helpers.waitForTransaction(voteResponseHash, provider)
 
-          get()[sliceKey].setStateByKey('executeTxMapper', {
-            ...get()[sliceKey].executeTxMapper,
-            [voteIdKey]: {
-              status: 'SUCCESS',
-              hash: voteResponseHash,
-              txLink: networks[1].scanTxPath(voteResponseHash),
-              error: null,
-            },
-          })
-
           dismissDeploying()
           const successNotificationMessage = t`Proposal executed successfully!`
           notifyNotification(successNotificationMessage, 'success', 15000)
+
+          // update proposal executed status
+          const response = await fetch(
+            `https://prices.curve.fi/v1/dao/proposals/details/${voteType.toLowerCase()}/${voteId}?tx_hash=${voteResponseHash}`,
+          )
+          const data: PricesProposalResponse = await response.json()
+
+          if ('detail' in data) {
+            return
+          }
```

Comment:
What does this mean? Maybe a comment would be nice

Reply:
sure, when the is no data the api returns an object with only a "detail" field